src/components/ecl-breadcrumb/ecl-breadcrumb.tsx

```diff
@@ -17,7 +17,7 @@ export class EclBreadcrumb {
   @Element() el: HTMLElement;
   @Prop() styleClass: string = '';
   @Prop() variant: string = 'default';
-  @Prop() theme: string = 'ec';
+  @Prop() theme: string;
```

Comment:
We need to define this prop as mutable, if you run the test you will see the reason why, see the other examples where this was updated.